.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: node_js
+node_js:
+  - node
+script:
+  - npm run lint
+  - npm test
+  - npm run typescript
+  - npm run build
+  - echo "Folder name in Digital Ocean Spaces - $TRAVIS_COMMIT"
+  - mkdir -p dist/@openmrs/esm-primary-navigation/$TRAVIS_COMMIT
+  - mv dist/*.* dist/@openmrs/esm-primary-navigation/$TRAVIS_COMMIT/
```

Comment:
change `esm-primary-navigation` to reflect the modules name

Reply:
Did that, and also removed the "after deploy" step to update import_map. Which I think is the right thing to do?